invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts

```diff
@@ -229,5 +229,21 @@ export const buildLinearTextToImageGraph = (
     addWatermarkerToGraph(state, graph);
   }
 
+  if (
+    !state.system.shouldUseNSFWChecker &&
+    !state.system.shouldUseWatermarker
+  ) {
+    graph.edges.push({
+      source: {
+        node_id: METADATA_ACCUMULATOR,
+        field: 'metadata',
+      },
+      destination: {
+        node_id: LATENTS_TO_IMAGE,
+        field: 'metadata',
+      },
+    });
+  }
```

Comment:
Really shouldn't this always be added, and the watermarker and nsfw functions _not_ add it?

Reply:
I guess its more complex than that though

Reply:
The watermarker and nsfw add it .. so I'm checking if those arent enabled. If they arent, then we add it. Maybe there's a better way of doing this.

Reply:
its fine to just write the metadata for latents to image, simpler for now, pr updated